src/makeInfinityAdder.js

```diff
@@ -5,6 +5,22 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
```

Comment:
The check `if (num === undefined)` is correct for determining when to return the sum. However, consider using `arguments.length === 0` to check if no arguments were passed, which is more explicit and handles cases where `undefined` might be passed intentionally.